package.json

```diff
@@ -25,7 +25,7 @@
     "cable_ready": "5.0.1",
     "debounced": "^0.0.5",
     "flatpickr": "^4.6.9",
-    "foundation-sites": "^5.5.2",
+    "foundation-sites": "^5.5.3",
```

Comment:
Can we lock the version to 5.X.X?

Reply:
Probably, but do you think we should do this update? (and any other highly unlikely updates to v5?)

Reply:
Why not? It would remove the question of updating in the future if we are on the latest already. :smile: 

But as you say, it's unlikely that there will be more. So no big impact either way. But we may as well finish this and lock the version in and then nobody will question it again, I guess.